README.md

```diff
@@ -573,8 +615,8 @@ function* test() {
 * Return address not found in frame
 * Should not directly enter OSR-compiled function
 * Sloppy function expects JSReceiver as receiver.
-* Smi addition overflow
-* Smi subtraction overflow
+* ~~Smi addition overflow~~
```

Comment:
I feel I am lost here. Does strike-out stand for "no longer present in V8" or "covered within README.md"? :)